tests/test_orm.py

```diff
@@ -60,8 +61,8 @@ def test_make_new_db(tmp_path: str) -> None:
     tmp_db.unlink()
 
 
-def test_make_and_populate_new_db(tmp_path: str) -> None:
-    """Confirm can create a new empty database, add genomes, close, and reopen."""
+def test_make_and_populate_comparisons(tmp_path: str) -> None:
+    """Populate new DB with config, genomes, and comparisons."""
     DUMMY_ALIGN_LEN = 400  # noqa: N806
```

Comment:
Could turn these dummy values into a fixture?

Reply:
I wasn't intending to use them beyond this localised test.